tests/plotting/test_plotting.py

```diff
@@ -3591,6 +3591,7 @@ def test_plot_texture_flip_y(texture):
 
 @pytest.mark.needs_vtk_version(9, 2, 0)
 @pytest.mark.skipif(CI_WINDOWS, reason="Windows CI testing segfaults on pbr")
+@pytest.mark.skipif(pv.vtk_version_info >= (9, 3), reason="This is broken on VTK 9.3")
```

Comment:
This isn't broken per-se but rather I think something between VTK 9.3 and the version of libosmesa shipped with it is causing chaos